tmva/tmva/src/CrossValidation.cxx

```diff
@@ -73,7 +75,7 @@ void TMVA::CrossValidationResult::Fill(CrossValidationFoldResult const & fr)
    UInt_t iFold = fr.fFold;
 
    fROCs[iFold] = fr.fROCIntegral;
-   fROCCurves->Add(static_cast<TGraph *>(fr.fROC.Clone()));
+   fROCCurves->Add(dynamic_cast<TGraph *>(fr.fROC.Clone()));
```

Comment:
@ashlaban what is the advantage of using dynamic cast when then we do not check if its outcome is nullptr?

Reply:
That would be that downstream code fail with a nullptr exception. Admittedly, this change might be unnecessary, since if `Clone()` is broken we have bigger problems.

However, since this is in a non-performance critical section I decided to err on the side of caution and abide by the heuristic "use `dynamic_cast`'s for downcasting".